src/devices/Qmc5883l/Mode.cs

```diff
@@ -0,0 +1,29 @@
+﻿// Licensed to the .NET Foundation under one or more agreements.
+// The .NET Foundation licenses this file to you under the MIT license.
+using System;
+
+namespace Iot.Device.Qmc5883l
+{
+    /// <summary>
+    /// Modes of Qmc5883l
+    /// </summary>
+    [Flags]
+    public enum Mode : byte
```

Comment:
could you convert this into bool and call it just `StandbyMode` or `LowPowerMode`?